core/src/core/block.rs

```diff
@@ -429,12 +438,12 @@ impl Block {
 	fn verify_coinbase(&self, secp: &Secp256k1) -> Result<(), secp::Error> {
 		let cb_outs = self.outputs
 			.iter()
-			.filter(|out| out.features.intersects(COINBASE_OUTPUT))
+			.filter(|out| out.features.contains(COINBASE_OUTPUT))
```

Comment:
`intersect` here will not work as expected if we ever compare against a bitflag that consists of more than a single bit - we need it to be a full `contains` to match correctly.